### PR TITLE
i18n(pt-BR): Update `guides/typescript.mdx` translation

### PR DESCRIPTION
**Pull request by viniciusdeliz**
#### Description (required)

Update the translation of i18n/guides/typescript.mdx

### CODE REVIEW

File: src/content/docs/pt-br/guides/typescript.mdx [hunk 1]
```diff
@@ -33,7 +35,12 @@ Adicionalmente, nossos templates incluem um arquivo `env.d.ts` dentro do diretó
 /// <reference types="astro/client" />
 ```
 
-Se você não está usando VSCode, você pode instalar o [plugin Astro TypeScript](https://www.npmjs.com/package/@astrojs/ts-plugin) para suportar a importação de arquivos `.astro` de arquivos `.ts` (que pode ser útil para reexportação).
+### Plugin editor TypeScript
```
Comment (yanthomasdev) [inline src/content/docs/pt-br/guides/typescript.mdx:38]: Acredito que aqui dê para melhorar a tradução do título, este é um plugin TypeScript para checar arquivos Astro em editores de código, acredito que assim ficaria mais compreensível:
```suggestion
### Plugin TypeScript para editores
```

**Review by yanthomasdev — CHANGES_REQUESTED** [review]
Good work @viniciusdeliz, I have a few suggestions here!

Also, I can see we're missing the changes of a few commits:
- https://github.com/withastro/docs/commit/6e5e00f9534b70217b74988d3f6d0f98f2772661 
- https://github.com/withastro/docs/commit/82677457dd9189e501cb8c0191c1693f32cb11f6
- https://github.com/withastro/docs/commit/3dd99e0401fe02855283fea0228e03e0d2a63485

Could you update your translation to include them? 🙌

**Comment by viniciusdeliz** [comment]
> Good work @viniciusdeliz, I have a few suggestions here!
> 
> Also, I can see we're missing the changes of a few commits:
> 
> * [6e5e00f](https://github.com/withastro/docs/commit/6e5e00f9534b70217b74988d3f6d0f98f2772661)
> * [8267745](https://github.com/withastro/docs/commit/82677457dd9189e501cb8c0191c1693f32cb11f6)
> * [3dd99e0](https://github.com/withastro/docs/commit/3dd99e0401fe02855283fea0228e03e0d2a63485)
> 
> Could you update your translation to include them? 🙌

Great review @yanthomasdev, I'm sorry for missing those.
Let me know if the file still requires changes and I'll promptly fix it.

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work!

**Pull request merged**